UserInterface/SearchBarDemos/SearchBarDemos/SearchBarDemos/MainPage.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:local="clr-namespace:SearchBarDemos"
+             mc:Ignorable="d"
+             x:Class="SearchBarDemos.MainPage"
+             Padding="10"
+             Title="SearchBar Demos">
+
+    <TableView Intent="Menu">
+        <TableRoot>
+            <TableSection>
+                <TextCell Text="XAML SearchBar"
+                          Detail="XAML SearchBar using an event handler to perform search."
```

Comment:
Might be best to shorten all the `Detail` text on this page. It's not fully displaying in my iOS simulator (depends which phone size is used). Personally, I'd lose the duplicate text (XAML SearchBar), as it's already in the `Text` property. So `Detail` would become "Use an event handler to perform search.".

Ditto for all the `Detail` properties on this page.